examples/crm/src/sales/TransferAdminRoleModal.tsx

```diff
@@ -0,0 +1,114 @@
+import Alert from '@mui/material/Alert';
+import Autocomplete from '@mui/material/Autocomplete';
+import Button from '@mui/material/Button';
+import CircularProgress from '@mui/material/CircularProgress';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import Skeleton from '@mui/material/Skeleton';
+import { default as MuiTextField } from '@mui/material/TextField';
+import { useState } from 'react';
+import {
+    useDataProvider,
+    useGetIdentity,
+    useGetList,
+    useRefresh,
+} from 'react-admin';
+import { useNavigate } from 'react-router';
+import { USER_STORAGE_KEY } from '../authProvider';
+import { CustomDataProvider } from '../dataProvider';
+import { Sale } from '../types';
+type TransferAdminRoleModalProps = {
+    open: boolean;
+    onClose(): void;
+};
+
+export function TransferAdminRoleModal({
+    open,
+    onClose,
+}: TransferAdminRoleModalProps) {
+    const { identity } = useGetIdentity();
+    const dataProvider = useDataProvider<CustomDataProvider>();
+    const refresh = useRefresh();
+    const navigate = useNavigate();
+    const {
+        data: sales,
+        error,
+        isPending,
+    } = useGetList('sales', {
+        filter: { administrator: false },
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** Starting from scratch, if I create my admin and then a new sale, I can't transfer my role, the selection is empty. I think it's linked to the fact that the administrator status is not well defined at creation.‏